README.md

```diff
@@ -166,28 +166,39 @@ deploy:
 1. Install arduino-cli from here: https://arduino.github.io/arduino-cli/installation/
 2. Download ci-arduino
    * `git clone https://github.com/adafruit/ci-arduino`
-3. Put these lines at the end of your `.bashrc` or `.bash_profile` if you're on OSX. Make sure to fill in the path to where you installed ci-arduino and replacing USER with your username.
+3. Put these lines at the end of your `.bashrc` or `.bash_profile` if you're on OSX. Make sure to fill in the path to where you installed ci-arduino and replacing USER with your username. Make sure to that the libraries directory exists by installing a library first:
+   * `arduino-cli lib install "Adafruit NeoPixel"` 
+   * Linux
    ```bash
    alias test-platforms='python3 ~/path/to/ci-arduino/build_platform.py'
-   export HOME=/home/USER/
+   export HOME=/home/USER
    ```
+   * Mac/OSX
+   ```bash
+   alias test-platforms='python3 ~/path/to/ci-arduino/build_platform.py'
+   export HOME=/Users/USER
```

Comment:
Is there a way to do this without exporting the `HOME` alias? Could you try making another alias? Exporting `HOME` ended up seriously messing up my home directory in macOS, very difficult to restore back to original alias.

Reply:
Ooh, yeah I did not think about that. I'll change the name so it won't conflict with that.

Reply:
Please do!